README.md

```diff
@@ -1,2 +1,179 @@
+```
+
+To watch and output to directories by using folder paths as your input and output
+
+### Install React Router
+co
+- To use the Router component provided to wrap your components and associate them with a path.
+
+```bash
+$ npm install react-router-dom 
+```
+### Install React Redux Loading Bar
+
+- To use the Router component provided to wrap your components and associate them with a path.
+
+```bash
+$ npm install --save react-redux-loading-bar
+```
+
+
+### Learn More
+ 
+  > This command will download all the dependancies of the project
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+
+## Run Tests
+
+ npm i @testing-library/jest-dom   
+ npm install --save-dev @testing-library/jest-dom
+
+```bash
+$ npm install @testing-library/react react-test-renderer jest-dom --save-dev
+```
+
+
+### Learn More
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+For tracking linter errors locally you need to follow these steps:
+
+
+- For tracking the linter errors in HTML file run:
+
+To learn React, check out the [React documentation](https://reactjs.org/).
+
+### Code Splitting
+- For tracking the linter errors in CSS file run:
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+  > `npx stylelint "\*_/_.{css,scss}"`
+### Analyzing the Bundle Size
+- And For tracking the linter errors in JavaScript file run:
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+  > `npx eslint .`
+### Making a Progressive Web App
+- Use `npm run build` to run the app in live server
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+
+### Deployment
+
+- `npm run build` fails to minify
+### Show your support
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+## Author
+
+👤 **Evans Sitibekiso**
+
+- GitHub: [@evans22j](https://github.com/evans22j)
+- Twitter: [@Evans_22J](https://twitter.com/Evans_22J)
+- LinkedIn: [@Evans Sitibekiso](https://www.linkedin.com/in/evans-sitibekiso/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+Give a :star: if you like this project! ❤️❤️❤️
+
+## Acknowledgments
+
+- Thanks to microverse for this project.
+
+## 📝 License
+
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) licensed.
```

Comment:
OPTIONAL
_This is not mentioned in the last review but this is a requirement so please add it._
- [ ] You should add a creative common license. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)